src/ccf/calc_ccf.jl

```diff
@@ -281,6 +281,38 @@ function ccf_1D(λ::A2, flux::A3, #line_list::ALL, #mask_shape1::A3
     return ccf_out
 end
 
+"""
+    `ccf_1D( λs, fluxes; ccf_plan )`
```

Comment:
Since it sounds like you're planning to call this directly, are you happy with this name?  
It's not really parallel with the names elsewhere (which tend to begin with verbs for functions that do significant work).  If we're going to change it, better sooner than later.

Reply:
I'm not unhappy with it, although I think best practices dictate that it should be parallel. Perhaps just `calc_ccf_1D`?

Reply:
Do we anticipate something other than 1D?
@alexander-wise Thoughts?
If we change it, it should be a separate PR anyway.

Reply:
I would call it calc_ccf_timeseries_1D or simply calc_ccf_timeseries to be in line with our other names. Also this should go in the "convenience" folder with other functions like it.